nsv13/code/modules/clothing/custom_clothes.dm

```diff
@@ -249,6 +249,15 @@
 	item_state = "sun"
 	dynamic_hair_suffix = ""
 
+/obj/item/clothing/accessory/bomber_jacket_accessory
+	name = "thin bomber jacket"
+	desc = "A cheap, wafer thin replica of a bomber jacket, this would fit on top of most things."
+	icon = 'nsv13/icons/obj/clothing/accessories.dmi'
+	icon_state = "bomberthin"
```

Comment:
```suggestion
	alternate_worn_icon = 'nsv13/icons/mob/custom_accessories.dmi'
	icon = 'nsv13/icons/obj/clothing/accessories.dmi'
	icon_state = "bomberthin"
```
Please do this and then add the worn sprite to a new file in that destination with the right icon state please.

Reply:
I can, but this turns it invisible...

Reply:
Alright nevermind then, I'll note it down to be fixed later.